McCorrections/src/LeptonWeights.cpp

```diff
@@ -114,7 +117,7 @@ double LeptonWeights::GetTriggerWeight(EventInfoBase& eventInfo) const
     const double eff_data = GetTriggerEfficiency(eventInfo, true);
     const double eff_mc = GetTriggerEfficiency(eventInfo, false);
     if(eff_mc == 0) {
-        if(eff_data != 0)
+        if(round(eff_data) != 0)
```

Comment:
don't use round here.

Reply:
There was an event with eff_data something like 0.00026 when the eff_mc = 0, that's why I use round, what should I do?

Reply:
In theory, if everything is applied correctly, this should not happen. Please, implement an extended exception message here and see what is going on.